stddef.h is missing in bcc_elf.h?

The following code failed to compile on Ubuntu 18.04. (This is what CMake does to check whether the symbol exists.)

```c
#include <bcc/bcc_elf.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef bcc_elf_foreach_sym
  return ((int*)(&bcc_elf_foreach_sym))[argc];
#else
  (void)argc;
  return 0;
#endif
}
```

```sh
% gcc a.c -lbcc
In file included from a.c:2:0:
/usr/include/bcc/bcc_elf.h:42:59: error: expected ‘)’ before ‘uint64_t’
 typedef int (*bcc_elf_symcb_lazy)(size_t, size_t, size_t, uint64_t, uint64_t,
                                                           ^~~~~~~~
/usr/include/bcc/bcc_elf.h:65:48: error: unknown type name ‘bcc_elf_symcb_lazy’; did you mean ‘bcc_elf_symcb’?
 int bcc_elf_foreach_sym_lazy(const char *path, bcc_elf_symcb_lazy callback,
                                                ^~~~~~~~~~~~~~~~~~
                                                bcc_elf_symcb
/usr/include/bcc/bcc_elf.h:80:42: error: unknown type name ‘size_t’; did you mean ‘__ssize_t’?
 int bcc_elf_symbol_str(const char *path, size_t section_idx,
                                          ^~~~~~
                                          __ssize_t
/usr/include/bcc/bcc_elf.h:81:24: error: unknown type name ‘size_t’; did you mean ‘__ssize_t’?
                        size_t str_table_idx, char *out, size_t len,
                        ^~~~~~
                        __ssize_t
/usr/include/bcc/bcc_elf.h:81:57: error: unknown type name ‘size_t’; did you mean ‘__ssize_t’?
                        size_t str_table_idx, char *out, size_t len,
                                                         ^~~~~~
                                                         __ssize_t
```

If I include "stddef.h", the program compiles. So shouldn't bcc_elf.h include stddef.h? Or am I missing something?

(It seems due to this issue, bpftrace's CMake failed to find [bcc_elf_foreach_sym](https://github.com/iovisor/bpftrace/blob/fc2af891503061708e455f8de0d89c96554abd4d/CMakeLists.txt#L72))

----
Environtment
- Ubuntu 18.0.4 (Linux 5.0.0-32generic)
- gcc-7

--- a/src/cc/bcc_elf.h
+++ b/src/cc/bcc_elf.h
@@ -20,6 +20,7 @@
 extern "C" {
 #endif
 
+#include <stddef.h>
 #include <stdint.h>
 
 struct bcc_elf_usdt {

